profiling/src/api.rs

```diff
@@ -153,6 +159,15 @@ impl std::fmt::Display for UpscalingInfo {
                 "Poisson = sum_value_offset: {}, count_value_offset: {}, sampling_distance: {}",
                 sum_value_offset, count_value_offset, sampling_distance
             ),
+            UpscalingInfo::PoissonNonSampleTypeCount {
+                sum_value_offset,
+                count_value,
+                sampling_distance,
+            } => write!(
+                f,
+                "Poisson = sum_value_offset: {}, count_value: {}, sampling_distance: {}",
+                sum_value_offset, count_value, sampling_distance
+            ),
```

Comment:
Clippy would probably suggest this:

```suggestion
            } => write!(
                f,
                "Poisson = sum_value_offset: {sum_value_offset}, count_value: {count_value}, sampling_distance: {sampling_distance}"
            ),
```

(or something close, writing code in a text box in a browser is hard)

This kind of thing is repeated a few times in this file (in and out of the PR).

Reply:
Interestingly clippy does not complain. I'd tend keep it as is for consistency reasons and clean up al of this in another PR, WDYT?

Reply:
See https://github.com/DataDog/libdatadog/pull/913